src/pages/user-profile/UserProfile.jsx

```diff
@@ -53,15 +53,12 @@ const UserProfile = () => {
   }
 
   const isTutor = preferredRole === UserRoleEnum.Tutor
-  const shouldShowPresentation =
-    (isTutor && isMyProfile) ||
-    (!isTutor && response.videoLink?.student) ||
-    (!isMyProfile && response.videoLink?.tutor)
+
```

Comment:
I added a space before VideoRresentationComponent because there wasn't any) it now looks like this:

![Screenshot 2024-08-15 113104](https://github.com/user-attachments/assets/3b84c938-3055-4be6-a1ac-f81f20e7dcf8)